make/docker-compose.tpl

```diff
@@ -94,37 +94,12 @@ services:
       options:  
         syslog-address: "tcp://127.0.0.1:1514"
         tag: "postgresql"
-  adminserver:
-    image: goharbor/harbor-adminserver:__version__
-    container_name: harbor-adminserver
-    env_file:
-      - ./common/config/adminserver/env
-    restart: always
-    cap_drop:
-      - ALL
-    cap_add:
-      - CHOWN
-      - SETGID
-      - SETUID
-    volumes:
-      - /data/config/:/etc/adminserver/config/:z
-      - /data/secretkey:/etc/adminserver/key:z
-      - /data/:/data/:z
-    networks:
-      - harbor
-    dns_search: .
-    depends_on:
-      - log
-    logging:
-      driver: "syslog"
-      options:  
-        syslog-address: "tcp://127.0.0.1:1514"
-        tag: "adminserver"
   core:
     image: goharbor/harbor-core:__version__
     container_name: harbor-core
     env_file:
       - ./common/config/core/env
+      - ./common/config/adminserver/env
```

Comment:
can they be merged?

Reply:
Because merge this file to core/env involve prepare script, Dengqian is working on it
Renamed to ./common/config/core/config_env in next PR.